api/configuration/v1alpha1/konnect_controlplaneref_types.go

```diff
@@ -41,4 +41,8 @@ type KonnectNamespacedRef struct {
 
 	// TODO: Implement cross namespace references:
 	// https://github.com/Kong/kubernetes-configuration/issues/36
+	// Currently only cluster scoped resources (KongVault) are allowed to set `konnectNamespacedRef.namespace`.
+	// Namespace is the namespace where the Konnect Control Plane is in.
+	// +optional
```

Comment:
Reword the comment to not include the TODO in the resulting CRDs.

```suggestion
	// TODO: Implement cross namespace references:
	// https://github.com/Kong/kubernetes-configuration/issues/36

	// Namespace is the namespace of the Konnect Control Plane.
	// Currently only cluster scoped resources (KongVault) are allowed to set `konnectNamespacedRef.namespace`.
	//
	// +optional
```